fluentui_progress/src/main/java/com/microsoft/fluentui/tokenized/progress/ProgressText.kt

```diff
@@ -0,0 +1,137 @@
+package com.microsoft.fluentui.tokenized.progress
+
+import androidx.compose.animation.core.LinearOutSlowInEasing
+import androidx.compose.animation.core.animateFloatAsState
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.*
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.BasicText
+import androidx.compose.material.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.StrokeCap
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.LayoutDirection
+import androidx.compose.ui.unit.dp
+import com.microsoft.fluentui.icons.ProgressTextIcons
+import com.microsoft.fluentui.icons.progresstexticons.DismissCircle
+import com.microsoft.fluentui.progress.R
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.token.ControlTokens
+import com.microsoft.fluentui.theme.token.controlTokens.ProgressTextInfo
+import com.microsoft.fluentui.theme.token.controlTokens.ProgressTextTokens
+import com.microsoft.fluentui.util.dpToPx
+
+/**
+ * ProgressTexts are used to give information through a text about the current progress.
+ * A ProgressText consists of a Text and a progressbar.
+ * @param text Text or info to display
+ * @param progress Progress of the progress indicator. 0.0 represents no progress and 1.0 represents full progress.
+ * @param onCancelClick onClick method to execute when cancel button is clicked
+ * @param modifier Modifier for the progress text
+ * @param progressTextTokens Token values for the ProgressText
+ */
+@Composable
+fun ProgressText(
+    text: String,
+    progress: Float,
+    onCancelClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    progressTextTokens: ProgressTextTokens? = null
+) {
+    val tokens = progressTextTokens
+        ?: FluentTheme.controlTokens.tokens[ControlTokens.ControlType.ProgressText] as ProgressTextTokens
+    val progressTextInfo = ProgressTextInfo()
```

Comment:
Take Progress as parameter in ProgressTextInfo()

Reply:
Could be a good idea if colors are to be changed with respect to progress.